src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/media/fixtures.js

```diff
@@ -0,0 +1,105 @@
+export const CPSMediaBlock = {
+  id: 'p07jl3lv',
+  subType: 'clip',
+  format: 'video',
+  title: 'Why dis pikin get fish skin',
+  synopses: {
+    short: 'Skin Condition: Why dis girl get Lamela Ichthyosis',
+    long:
+      'Producers: Gift Andrew, Dan Ikpoyi\n\nDis na tori of little Amida wey get skin condition wey no dey common wey doctors say dem neva find di cure all ova di world.\n\nDi condition make her bodi dey old like old pesin and e don affect how she dey grow.\n\nIf pesin see five year old Amida, e go think say na two years old pikin sake of say di disease don make am look so small.\n\nNa Gbajare crew for Radio Nigeria, Ibadan an im first discover Amida and her condition.\n\nDi name of di condition na Lamellar Ichthyosis and na genetic defect dey cause am, e dey make pesin skin dey like fish body, according to Skin Doctor Ayesha Akinkugbe wey yan wit BBC Pidgin.\n\n“Di condition dey cause bad  odour and plenti pipo fit dey stigmatize pipo wey dey suffer from am, so dem suppose get plenti support from parents and family” na so Doctor Ayesha tok.\n\nAmida mama wey be hairdresser say di small girl no gree go school again sake of say her fellow students dey laugh her for school.\n\n"My stormach dey boil like hot water wen I bin cari her belle" na so Amida mama tok.\n“We don spend plenti money to cure am and till now no cure”  She add.\n\nFive year old Amida wey dey live wit her mama for Isheyin, Oyo State South West Nigeria go need plenti support to survive dis condition.',
```

Comment:
Can you shorten this please, all this text isn’t needed

Reply:
👍 shortened